### DISCUSSION TITLE
Grading *without* autograder?

### DISCUSSION THREAD

**Discussion by pdeffebach** [community/community#38176] [GitHub Education]
Is there a way to keep track of grades in github classroom without using the auto-grader? I just want to read a student's assignment and give them a score from 1-10. I want to be able to click "download grades" from the classroom page and keep track of those not-auto-generated scores.

**Comment by ossanna16** [comment]
Hi there @pdeffebach and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
